stubs/PyYAML/yaml/nodes.pyi

```diff
@@ -1,31 +1,37 @@
-from typing import Any
+from typing import Any, ClassVar
+
+from yaml.error import Mark as Mark
 
 class Node:
-    tag: Any
+    tag: str
     value: Any
-    start_mark: Any
-    end_mark: Any
-    def __init__(self, tag, value, start_mark, end_mark) -> None: ...
+    start_mark: Mark | Any
```

Comment:
I am following @Akuli's advice here to avoid optional attributes ([ref](https://github.com/python/typeshed/pull/5627#discussion_r650521022))